src/core/index.ts

```diff
@@ -277,3 +277,9 @@ export {
   MiddleTextTruncation,
 } from './utils';
 export type { ThemeType, MiddleTextTruncationProps } from './utils';
+
+export { SkipToContentLink, SkipToContentMarker } from './SkipToContent';
```

Comment:
This should be in alphabetical order see [here](https://github.com/iTwin/iTwinUI-react/blob/main/CONTRIBUTING.md?plain=1#L98)

Reply:
Fixed